lib/settingsGenerated.ts

```diff
@@ -144,4 +144,5 @@ export const defaultSettings: ISettings = {
":{"server":{"type":"string","enum":["off","messages","verbose"]}},"additionalProperties":false},"paths":{"type":"object","properties":{"additional":{"type":"array","items":{"type":"string"}},"libraryMapFiles":{"type":"array","items":{"type":"string"}},"ignoreRegex":{"type":"string"},"ignoreFiles":{"type":"array","items":{"type":"string"}}},"additionalProperties":false},"analysis":{"type":"object","properties":{"verilogAnalysis":{"type":"boolean"},"conditionalAnalysis":{"type":"object","properties":{},"patternProperties":{"":{"type":"string"}}}},"additionalProperties":false},"semanticTokens":{"type":"boolean"},"semanticTokensDirectionColoring":{"type":"boolean"},"rules":{"type":"object","properties":{"coding-style":{"type":"boolean"},"consistent-casing":{"type":"boolean"},"component":{"type":"boolean"},"instantiation":{"type":"boolean"},"configuration":{"type":"boolean"},"multiple-definition":{"type":"boolean"},"not-declared":{"type":"boolean"},"naming-style":{"type":"boolean"},"casing-style":{"type":"boolean"},"type-checking":{"type":"boolean"},"type-resolved":{"type":"boolean"},"unused":{"type":"boolean"},"use-clause":{"type":"boolean"},"empty":{"type":"boolean"},"constant-write":{"type":"boolean"},"parser":{"type":"boolean"},"unit":{"type":"boolean"},"not-allowed":{"type":"boolean"},"order":{"type":"boolean"},"attribute":{"type":"boolean"}},"additionalProperties":false},"style":{"type":"object","properties":{"preferredLogicTypePort":{"type":"string","enum":["unresolved","resolved","ignore"]},"preferredLogicTypeSignal":{"type":"string","enum":["unresolved","resolved","ignore"]},"preferredLogicTypeRecordChild":{"type":"string","enum":["unresolved","resolved","ignore"]},"ieeeCasing":{"type":"string","enum":["lowercase","UPPERCASE"]},"unusedPrefix":{"type":"string"},"unusedSuffix":{"type":"string"},"signalPrefix":{"type":"string"},"signalSuffix":{"type":"string"},"variablePrefix":{"type":"string"},"variableSuffix":{"type":"string"},"objectCasing":{"type":"string","enum":["snake_case","PascalCase","camelCase","CONSTANT_CASE","ignore"]},"constantGenericCasing":{"type":"string","enum":["snake_case","PascalCase","camelCase","CONSTANT_CASE","ignore"]},"constantPrefix":{"type":"string"},"constantSuffix":{"type":"string"},"genericPrefix":{"type":"string"},"genericSuffix":{"type":"string"},"outPrefix":{"type":"string"},"outSuffix":{"type":"string"},"inPrefix":{"type":"string"},"inSuffix":{"type":"string"},"inoutPrefix":{"type":"string"},"inoutSuffix":{"type":"string"},"parameterOutPrefix":{"type":"string"},"parameterOutSuffix":{"type":"string"},"parameterInPrefix":{"type":"string"},"parameterInSuffix":{"type":"string"},"parameterInoutPrefix":{"type":"string"},"parameterInoutSuffix":{"type":"string"},"labelCasing":{"type":"string","enum":["lowercase","UPPERCASE","ignore"]},"instantiationLabelPrefix":{"type":"string"},"instantiationLabelSuffix":{"type":"string"},"portOmission":{"type":"boolean"}},"additionalProperties":false}},"additionalProperties":false};
```

Comment:
do we need this here, can we not just load the json? I would really try to reduce the amount of generated junk in the git.